src/nacl/public.py

```diff
@@ -154,6 +154,9 @@ def generate(cls):
         return cls(random(PrivateKey.SIZE), encoder=encoding.RawEncoder)
 
 
+_Box = TypeVar("_Box", bound="Box")
+
+
```

Comment:
This should have been added in the following commit; apologies for the poor hygiene here.